Software/Python/setup.py

```diff
@@ -44,7 +44,7 @@
 import setuptools
 setuptools.setup(
     name = "grovepi",
-    version = "1.0.1",
+    version = "1.0.0",
```

Comment:
why are we going back in versions?

Reply:
Well, we don't really do any versioning here so I thought "why not have a round number". What do you think?

I don't have any strong preference here.

Reply:
We already have a 1.0.0 out there and a 1.0.1, so no going back.
In fact I think the version on pip is 1.0.0 (I may be wrong, it's been years!)